<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1383</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>1061</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextEdit" name="txtEditor"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>520</y>
      <width>1061</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Consola</string>
        </attribute>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1031</width>
           <height>201</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Variables</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1031</width>
           <height>211</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1383</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionCrear"/>
    <addaction name="separator"/>
    <addaction name="actionAbrir"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar_Como"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuCompilar">
    <property name="title">
     <string>Compilar</string>
    </property>
    <addaction name="actionCompilar"/>
   </widget>
   <widget class="QMenu" name="menuReportes">
    <property name="title">
     <string>Reportes</string>
    </property>
    <addaction name="actionErrores"/>
    <addaction name="separator"/>
    <addaction name="actionGenerador_AST"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuCompilar"/>
   <addaction name="menuReportes"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCrear">
   <property name="text">
    <string>Crear</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionGuardar_Como">
   <property name="text">
    <string>Guardar Como</string>
   </property>
  </action>
  <action name="actionErrores">
   <property name="text">
    <string>Errores</string>
   </property>
  </action>
  <action name="actionGenerador_AST">
   <property name="text">
    <string>Generador AST</string>
   </property>
  </action>
  <action name="actionCompilar">
   <property name="text">
    <string>Compilar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
